buyer-api/src/queues/transactionQueue.js

```diff
@@ -60,7 +60,8 @@ const enqueueTransaction = (account, signWith, params, gasPrice, options = {}) =
   const {
     name, priority = 1000, attempts = 20, backoffType = 'linear',
```

Comment:
Do we need to get the `name` from the `options` ?? Can we just use the `signWith`?

Reply:
I agree that `name` option is not needed. I will remove it. 
What do you think about using `signWith` param to identify the transaction? 🤔 
I now think that we should enqueue the desired transaction, and later determine which is the signing function to use. In other words, change the firm to the following:
```js
const enqueueTransaction = (account, name, params, gasPrice, options = {}) => { ... }
```

Reply:
Actually that makes more sense. Instead of sending a name and then tweaking its string to check what it should do (i.e. `signWith.replace('sign', '')`), we could send the desired name and then do whatever you need with it to parse/understand it and do the corresponding queue.

Reply:
Will take care of this in the next PR along with the priorities improvement